suite-common/wallet-utils/src/numberFormatUtils.ts

```diff
@@ -0,0 +1,15 @@
+export const getDecimalSeparator = (locale: string) => {
```

Comment:
those utils are not used or am I looking wrong?

Reply:
Actually, yes, I was trying one thing but ended up not needing them. Will remove :)

Reply:
97bf0112f9b74c51f4c29d88982cf9e0e3388d9c